src/components/time-entries/TimeEntries.tsx

```diff
@@ -29,6 +32,10 @@ export const TimeEntries = (props: TimeEntriesProps) => {
     removeTimeEntry(id);
   };
```

Comment:
To make full use of the useEffect hook, you use give it a second argument (default is []) which tells the useEffect when to fire its functions

Reply:
I have the default second argument, but GitHub made that part invisible in here :)